README.md

```diff
@@ -55,3 +55,57 @@ registry service like [quay.io](https://quay.io).
     ```shell
     make deploy IMG=<registry>/<imagename>:<tag>
     ```
+
+### Deploy Operator using OLM
+
+If you would like to deploy Operator using OLM, you'll need to build and push the bundle image and index image. You need to host the images on a public registry service like [quay.io](https://quay.io).
+
+1. Build your bundle image
+    ```shell
+    make bundle-build REPO=<registry>
+    ```
+1. Push the bundle image
+    ```shell
+    make docker-push IMG=<bundle image name>
+    ```
+1. Build the index image
+
+    OPM needs to be installed first. Get it from [here](https://github.com/operator-framework/operator-registry/releases).
+    ```shell
+    make bundle-index-build REPO=<registry>
+    ```
+1. Push the index image
+    ```shell
+    make docker-push IMG=<index image name>
+    ```
+1. Create the CatalogSource/OperatorGroup/Subscription
+    ```yaml
+    ---
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: CatalogSource
+    metadata:
+      name: gatekeeper-operator
+      namespace: openshift-marketplace
```

Comment:
Should this be `gatekeeper-system`? If not, can we avoid using `openshift`? `operator-sdk` installs OLM in the `olm` namespace and creates an `operators` namespace, so perhaps we can use `operators`.

Reply:
I changed namespace to gatekeeper-system for catalogsource and also update it in operatorgroup